app/assets/stylesheets/components/_card_home_page.scss

```diff
@@ -31,3 +32,10 @@
   align-items: flex-end;
   position: relative;
 }
+
+.credit-card .card-trip-user {
+  position: absolute;
+  right: 16px;
+  top: -20px;
+  width: 40px;
+}
```

Comment:
This was part of [LeWagon's card](https://uikit.lewagon.com/documentation#card_trip). It was left out somehow.